### DISCUSSION TITLE
[Need Help] Upgrading to 2.0.0-rc.8 brings some problems

### DISCUSSION THREAD

**Discussion by recoluan** [vuepress/core#1508] [Q&A] [answered]
**Problem:**

@meteorlxy @Mister-Hope 

When I upgraded to 2.0.0-rc.8, the resource load was pending after starting the service, and it could not be compiled. Please help me, thanks.

---

**Manifestation:**

![image](https://github.com/vuepress/core/assets/18067907/21b625cb-5fb7-4cbc-8835-647417c9c665)

![image](https://github.com/vuepress/core/assets/18067907/280690c7-08b8-4eab-a0fe-ecfce8caba7f)

---

**Recurrence**

- Code: https://github.com/vuepress-reco/vuepress-theme-reco
- Run: pnpm install && pnpm build && pnpm dev

**Comment by meteorlxy** [comment]
being busy these days, will take a look later

**Reply by recoluan** [reply]
Okay, looking forward to the replies.

**Reply by meteorlxy** [reply]
I took a brief look at your project and there's a bit too many hacks in it. I even got stuck on vite compiling 🤕 Really hard to debug

**Reply by recoluan** [reply]
I am now able to locate a compilation problem at present, apart from Layout.vue, for other layout components, as long as there is content in the <script> tag, it cannot be compiled and passed, and emptying the content in the <script> tag can compile normally.

**Reply by Mister-Hope** [reply]
What do you mean by there are content in script tag?

Provide some example codes, please

**Reply by recoluan** [reply, marked as answer]
I solved the problem with commit [fix: be available under webpack builder](https://github.com/vuepress-reco/vuepress-theme-reco/commit/b8dfab847c6c22054c03783c83c6e3d30b304ad1) and [fix: fix routing problem under webpack builder](https://github.com/vuepress-reco/vuepress-theme-reco/commit/12ccac897978bfa50318140078d1d7fe0aefb4fd).

**Comment by Mister-Hope** [comment]
busy for my postgraduate thesis either.
